bundles/org.openhab.binding.tasmotaplug/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="tasmotaplug"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Tasmota Plug Thing -->
+	<thing-type id="plug">
+		<label>Plug</label>
+		<description>
+			Tasmota Smart Plug
+		</description>
+
+		<channels>
+			<channel id="power" typeId="system.power"/>
+			<channel id="power2" typeId="system.power"/>
+			<channel id="power3" typeId="system.power"/>
+			<channel id="power4" typeId="system.power"/>
```

Comment:
Probably adding labels/descriptions should be added here in order to better distinguish the channels.

Reply:
I agree... I thought I was being efficient by using the system channel. I changed them to use channels defined here so the labels and descriptions can be specified.

Reply:
It was perfect fine to use the `system.power` channel type, I only meant to suggest that you could override the label/description:
```xml
<channel id="power" typeId="system.power">
    <label>Power</label>
    <description>Controls the smart plug relay for the 1st channel</description>
</channel>
```

Reply:
OK, I did not see an example of that at first. I have updated and tested using the overridden labels/descriptions so it should be good now.